Utilities/OtherClaimTypes.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Practice.Utilities
+{
+    public static class OtherClaimTypes
+    {
+        public static string Age => "Age";
```

Comment:
Please use `Const` instead `static` (with `get {}`) when we want define an unchangeable field.